src/patterns/check-answers/index.md.njk

```diff
@@ -38,14 +38,14 @@ However, if your service asks users for longer or more detailed answers, a full
 
 ### Make the page easy to understand
 
-On your check answers page you should:
+On your check answers page, you should:
 
-- make it clear there’s a task to perform on the page
+- use the page title to tell users what they have to do - otherwise, they may not know because the 'submit' button is at the end of the page
```

Comment:
How about this?
- uses 'might' instead of 'may' (content team convention to avoid confusion with giving permission)
- uses 'the user' instead of users (I'm trying to be more consistent where we describe what's happening in an interaction)

```suggestion
- use the page title to tell the user what they need to do — otherwise, they might miss the submit button at the end of the page```

Reply:
@calvin-lau-sig7 That looks great - thanks!